students/14th/seungyunlee/lndex.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>repl.it</title>
+    <link rel="stylesheet" href="/style/style.css" />
+  </head>
+  <body>
+    <!-- 인스타 로고 ~ 비밀번호를 잊으셨나요? -->
+    <div class="mainbox">
+      <div class="mainbox__box1">
+        <!-- 입력창~ 비밀번호를 잊으셨나요? -->
+        <div class="main__top">
+          <h1 class="logo"><img src="./img/logo_text.png" alt="" /></h1>
```

Comment:
이렇게 줄 바꿔서 써주시는게 더 보기 좋습니다.
그리고 img 태그에서 이미지를 설명해주는
alt 속성이 src 속성보다 더 중요해서
alt 속성을 먼저 써주세요!
```suggestion
          <h1 class="logo">
            <img src="./img/logo_text.png" alt="" />
          </h1>
```